collect_app/src/main/res/values/strings.xml

```diff
@@ -12,20 +12,14 @@
 -->
 <resources xmlns:tools="http://schemas.android.com/tools" tools:locale="en"
            tools:ignore="MissingTranslation">
-<string name="accept_location">Record Location</string>
```

Comment:
@lognaturel I agree with you here is a fix https://github.com/opendatakit/collect/pull/1089
but if we use `Record Location` instead of `Start Geopoint` we should change `Start GeoTrace` to `Record Geotrace` and `Start GeoShape` to `Record GeoShape` I think. What is your opinion?